core/src/test/java/kafka/server/share/DelayedShareFetchTest.java

```diff
@@ -229,12 +233,12 @@ public void testReplicaManagerFetchShouldHappenOnComplete() {
         ShareFetchData shareFetchData = new ShareFetchData(
                 new FetchParams(ApiKeys.SHARE_FETCH.latestVersion(), FetchRequest.ORDINARY_CONSUMER_ID, -1, MAX_WAIT_MS,
                         1, 1024 * 1024, FetchIsolation.HIGH_WATERMARK, Optional.empty()), groupId, Uuid.randomUuid().toString(),
-                new CompletableFuture<>(), partitionMaxBytes);
+                new CompletableFuture<>(), partitionMaxBytes, 100);
```

Comment:
Can we use this 100 as a constant, I see its usage at many places in this file and other test files.

Reply:
Done.